tus-servers/file-server-https/readme.md

```diff
@@ -0,0 +1,38 @@
+## Purpose
+
+
+The purpose of this server is to be able to run the tus-client/broswer locally on https, and test APIM integration.
+This is a DEV only test for integration.
+
+
+## Create Cert for Local Dev 
+
+Create a folder __cert__, at the root of file-server-https.
```

Comment:
format request -  consider putting ticks around: `__cert__` and `file-server-https`